lib/l10n/intl_ja.arb

```diff
@@ -1,9 +1,9 @@
 {
-  "home_tab_top_anime": "トップアニメ",
+  "home_tab_top_anime": "講義を録音する",
```

Comment:
home_tab_top_anime is now "Record Lecture"? What happened to the "Top Anime" feature?